File: rb/test/exists_test.rb
# MbtaV3 SDK exists test

require "minitest/autorun"
require_relative "../MbtaV3_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = MbtaV3SDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- MbtaV3 SDK exists test

local sdk = require("mbta-v3_sdk")

describe("MbtaV3SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
